p4-16/psa/PSA.mdk

```diff
@@ -83,10 +83,10 @@ Table [#table-extern-usage]).
 [switch]: psa_pipeline.png { width: 100%; }
 
 A programmer targeting the PSA is required to instantiate objects for
-the programmable blocks that conform to these APIs. Note that the
-programmable block APIs are templatized on user defined headers and
-metadata. In PSA, the user can define a single metadata type for all
-controls.
+the programmable blocks that conform to these APIs (see section
+[#sec-programmable-blocks]).  Note that the programmable block APIs
+are templatized on user defined headers and metadata. In PSA, the user
+can define a single metadata type for all controls.
 
```

Comment:
Only change in this paragraph is adding the cross-reference.  Sorry, I re-formed the paragraph text and this diff doesn't make it obvious that everything else remained the same.